src/lua/elrsV2.lua

```diff
@@ -458,7 +458,7 @@ local function parseDeviceInfoMessage(data)
   end
   if deviceId == id then
     deviceName = devicesName
-    deviceIsELRS = fieldGetValue(data,offset,4) == 0x454C5253 -- SerialNumber = 'E L R S'
+    deviceIsELRS_TX = (fieldGetValue(data,offset,4) == 0x454C5253) == (deviceId == 0xEE) -- SerialNumber = 'E L R S' and ID is TX module
```

Comment:
should this be ` and ` not ` == `? Otherwise `false == false` would falsely enable deviceIsELRS_TX

Reply:
ah man .... just a simple check and I missed it. fixed it, thanks a lot cap!